docs/sidebars.js

```diff
@@ -172,6 +172,7 @@ module.exports = {
                     label: 'Architecture', // name still confusing with architecture page elsewhere
                     items: [
                         'arch-overview',
```

Comment:
the architecture overview will probably become outdated BTW. It's related to [this ticket](https://rasahq.atlassian.net/browse/ATO-483) but not quite the same

Reply:
yes, defintely, but we need a ticket in our backlog. Didn't have the time to create it yet

Reply:
Added [Adapt Architecture Overview page with user tailored content](https://rasahq.atlassian.net/browse/ATO-834)